documentation/api/query/v4/upgrading-from-v3-api.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "PuppetDB 2.3 » API » v4 » Upgrading to PuppetDB 3.0"
+layout: default
+canonical: "/puppetdb/latest/api/query/v4/upgrading-from-v3.html"
+---
+
+### Preparing for PuppetDB 3.0
```

Comment:
This title, and the document title, and the title in the nav are now different - did we want to fix this before the merge?